src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js

```diff
@@ -90,7 +90,7 @@ const DetailHeader = () => {
   const location = useLocation();
 
   const showSplitButton = useMemo(() => {
-    if (incident.status.state === 'o' || incident.status.state === 'a') return false;
+    if (['o', 'a', 's'].some(value => value === incident.status.state)) return false;
```

Comment:
I would use `includes`  instead of `some` here.

Reply:
Thanks, I did that first, but further in this file is this construction with `some` used so I prefer to keep it consistent with the rest of the code. Performance wise makes no difference.

Reply:
I'd use `includes` as well - if that's the initial thought for all 3 of us, it makes sense to use `includes` here and change the other occurance of `some` to `includes` as well